packages/components/src/button/style.scss

```diff
@@ -296,8 +291,6 @@
 		}
 
 		.dashicon {
-			display: inline-block;
-			flex: 0 0 auto;
 			margin-left: 2px;
 			margin-right: 2px;
 		}
```

Comment:
This left/right margin exists to widen the default size of the dash icon, 20px, to 24px.

As this comment suggests, ideally I would like to remove these margins as well, but so far I have not found an approach that makes sense.

One approach I thought of is to use padding instead of margins to widen the size and center the dash icon font of the pseudo-element inside.

```css
.dashicon {
	display: inline-flex;
	justify-content: center;
	align-items: center;
	padding: 2px;
	box-sizing: content-box;
}
```

Reply:
Yes, padding might be a little less fragile than margin.